nodejs/dist/uws.js

```diff
@@ -289,6 +289,20 @@ class Server extends EventEmitter {
         this._upgradeListener = null;
         this._noDelay = options.noDelay === undefined ? true : options.noDelay;
         this._lastUpgradeListener = true;
+        this._abortConnection = (socket, code, name) => {
+          try {
+            const response = [
+              'HTTP/1.1 ' + code + ' ' + name,
+              'Content-Type: text/html'
+            ];
+            socket.write(response.concat('', '').join('\r\n'));
```

Comment:
Can't this be made much simpler, something like

`socket.end('HTTP/1.1 ' + code + ' ' + name + '\r\n\r\n');`